src/index.html

```diff
@@ -1,13 +1,61 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/style.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product-card" data-qa="card">
+      <div class="product-card__content">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="product-card__image"
+        >
+
+        <div class="product-card__description">
+          <h3 class="product-card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+
+          <div class="product-card__code">Product code: 195434</div>
+
+          <div class="product-card__rate">
+            <div class="product-card-stars">
+
+              <div class="product-card-stars__star"></div>
+              <div class="product-card-stars__star"></div>
+              <div class="product-card-stars__star"></div>
+              <div class="product-card-stars__star"></div>
+              <div class="product-card-stars__star"></div>
+              
+            </div>
+
+            <div class="product-card__text">Reviews: 5</div>
+          </div>
+
+          <div class="product-card__price">
+            <div class="product-card__label">
+              Price:
+            </div>
+            <div class="product-card__cost">$2,199</div>
+          </div>
+        </div>
+
+        <a
+            href="#"
+            class="button"
+            data-qa="hover">
+          Buy
+        </a>
```

Comment:
```suggestion
        <a
          href="#"
          class="button"
          data-qa="hover"
        >
          Buy
        </a>
```
Pay attention to the formatting